sdk/core/Azure.Core.TestFramework/src/TestEnvironment.cs

```diff
@@ -356,10 +384,18 @@ private void EnsureValue(string name, string value)
         {
             if (value == null)
             {
-                var prefixedName = _prefix + name;
-                throw new InvalidOperationException(
-                    $"Unable to find environment variable {prefixedName} or {name} required by test." + Environment.NewLine +
-                    "Make sure the test environment was initialized using eng/common/TestResources/New-TestResources.ps1 script.");
+                string prefixedName = _prefix + name;
+                string message = $"Unable to find environment variable {prefixedName} or {name} required by test." + Environment.NewLine +
+                                 "Make sure the test environment was initialized using the eng/common/TestResources/New-TestResources.ps1 script.";
+                if (_bootstrappingException != null)
+                {
+                    message += Environment.NewLine + "Resource creation failed during the test run. Make sure PowerShell version 6 or higher is installed.";
```

Comment:
Why PowerShell 6? Seems like tight coupling with something else, much like checking if this is Windows. If using env vars, why wouldn't other platforms work?

Reply:
It says at least powershell 6. This is the same messaging we use in the contributing.md file.